index.js

```diff
@@ -17,6 +17,7 @@ var new_client = function(api_key, config) {
 
   client.base_uri = (config.base_uri || 'https://app.launchdarkly.com').replace(/\/+$/, "");
   client.stream_uri = (config.stream_uri || 'https://stream.launchdarkly.com').replace(/\/+$/, "");
+  client.events_uri = (config.events_uri || 'https://events.launchdarkly.com').replace(/\/+$/, "");
```

Comment:
Line is too long.<br>Identifier 'events_uri' is not in camel case.<br>Mixed double and single quotes.